### PR TITLE
Various PEP8 and style fixes

### PR DESCRIPTION
**Pull request by marcharper**

### CODE REVIEW

File: axelrod/mock_player.py [hunk 1]
```diff
@@ -17,7 +18,9 @@ class MockPlayer(Player):
 
     name = "Mock Player"
 
-    def __init__(self, actions: List[Action] =None, history: List[Action] =None, state_dist: defaultdict =None) -> None:
+    def __init__(self, actions: List[Action] = None,
+                 history: List[Action] = None,
+                 state_dist: defaultdict = None) -> None:
```
Comment (drvinceknight) [inline axelrod/mock_player.py]: My linter is picking `defaultdict = None` (etc) up:

```
[bad-whitespace] No space allowed around keyword argument assignment [python/pylint]
```

It's happy with:

```python
state_dist: defaultdict=None) -> None:
```

**Comment by marcharper** [comment]
For type hints there are supposed to be spaces, looking at the examples in [its PEP](https://www.python.org/dev/peps/pep-0484/), e.g.:

```
def func(x: Union[int, None, Empty] = _empty) -> int:
```
and
```
def embezzle(self, account: str, funds: int = 1000000, *fake_receipts: str) -> None:
```

**Comment by drvinceknight** [comment]
> For type hints there are supposed to be spaces, looking at the examples in its PEP, e.g.:

Cool. I wondered if it might be something like this. :+1:

**Comment by marcharper** [comment]
@drvinceknight Made some changes. Can you diagnose the travis issue with the docstring? I don't understand what it wants.

**Comment by drvinceknight** [comment]
I'll take a look. Note unit tests are also failing because of your attribute/method rename in dbs.

**Comment by marcharper** [comment]
Unittests fixed.

**Comment by drvinceknight** [comment]
Cool, I'm heading out for a walk but assume it's ok if I push a fix directly for the docstring later (I expect it's just sphinx being very picky about some indentation). :)

**Comment by marcharper** [comment]
> Cool, I'm heading out for a walk but assume it's ok if I push a fix directly for the docstring later (I expect it's just sphinx being very picky about some indentation). :)

Please do!

**Comment by drvinceknight** [comment]
Fixing the merge conflicts now :)

**Comment by drvinceknight** [comment]
Assuming tests pass etc, I am happy with this :+1:

**Pull request merged**